src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java

```diff
@@ -494,7 +495,16 @@ public enum MuleRuntimeFeature implements Feature {
    */
   DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION(
       "When enabled, implicit configuration for the XML SDK won't be created.",
-      "W-10634129", "4.7,0", MULE_DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION);
+      "W-10634129", "4.7,0", MULE_DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION),
+
+  /**
+   * When enabled, the MDC context will reset after XML SDK Operation has been executed.
+   *
+   * @since 4.8.0
```

Comment:
This one was ok as it was. The Feature Flag will be cherry picked, so it will exist in all those versions. It is only the default value that changes.
```suggestion
   *
   * @since 4.8.0, 4.7.1, 4.6.4, 4.4.0-202306
```